manifests/server/set_table_attribute.pp

```diff
@@ -0,0 +1,22 @@
+define postgresql::server::set_table_attribute (
```

Comment:
"table_attribute" would be a better name.  I don't think imperative is a good style on a declarative language.